api/tests/prescription/learner-management/acceptance/application/organization-import-route_test.js

```diff
@@ -11,7 +12,7 @@ import {
 describe('Acceptance | Application | organization-import', function () {
   let server;
 
-  beforeEach(async function () {
+  before(async function () {
```

Comment:
** suggestion **: je propose de garder le `createServer` dans une `beforeEach` pour éviter des effets de bord non voulus.

Reply:
Je ne vois pas l'effet de bord qu'il pourrait y avoir à ne créer qu'un `server`. Plutôt que de le lancer à chaque test ? 🤔

Reply:
le principe du test c'est de partir d'un environnement maitrisé et reproductible. Là on ne dé-enregistre pas les routes, donc selon comment se lance les tests il peut y avoir 1 ou plusieurs routes enregistrées

Reply:
Sur un test d'acceptance, le createServer enregistre toutes les routes automatiquement. 
Plutôt que de le faire à chaque it, le faire une seule fois me semblait judicieux 🤔 et totalement maitrisé.

Je pense qu'on sous évalue l'utilité du `before` et/ou qu'on a peur de l'utiliser pour ne pas induire en erreur. c'est dommage de devoir "perdre" du temps a chaque it pour créer un serveur alors que le before rempli le job.

Reply:
j'avoue, c'est plus l'eventualité du probleme qui me chiffonne mais on partir la dessus